starfish/core/segmentation_mask.py

```diff
@@ -101,6 +111,12 @@ def append(self, mask: xr.DataArray):
         validate_segmentation_mask(mask)
         self._masks.append(mask)
 
+        for axis in Axes:
+            if axis.value in mask.coords:
+                max_val = mask.coords[axis.value].values[-1]
```

Comment:
why not the last coords value?

Reply:
you mean `mask.coords[axis.value][-1]`? it's because xarray is weird so a singular value returned is still an xarray, so we access it via the numpy array instead